pkg/infrastructure/gcp/clusterapi/clusterapi.go

```diff
@@ -223,12 +223,21 @@ func (p Provider) InfraReady(ctx context.Context, in clusterapi.InfraReadyInput)
 func (p Provider) DestroyBootstrap(dir string) error {
 	logrus.Warnf("Destroying GCP Bootstrap Resources")
 	metadata, err := metadata.Load(dir)
+	ctx := context.Background()
```

Comment:
Nit: do this after check for err != nil ?

Reply:
Good catch. Fixed.